src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/FilterSet.cs

```diff
@@ -80,6 +81,7 @@ static FilterSet()
             PropertyFilter = CreateCompletionFilterAndAddToBuilder(FeaturesResources.Properties, 'p', WellKnownTags.Property);
             MethodFilter = CreateCompletionFilterAndAddToBuilder(FeaturesResources.Methods, 'm', WellKnownTags.Method);
             ExtensionMethodFilter = CreateCompletionFilterAndAddToBuilder(FeaturesResources.Extension_methods, 'x', WellKnownTags.ExtensionMethod);
+            OperatorFilter = CreateCompletionFilterAndAddToBuilder(FeaturesResources.Operators, 'r', WellKnownTags.Operator);
```

Comment:
@genlu no problem with this, right?

Reply:
Yup, this is fine, the shortcut (alt + r) is only relevant when completion list shows